src/components/DateField/Example.md

```diff
@@ -343,7 +343,7 @@ In `value` controlled mode
 const { DateField, StackPanel, StackItem, Button } = require('precise-ui');
 
 const Example = () => {
-  const [ value, setValue ] = React.useState('14-06-1988');
+  const [ value, setValue ] = React.useState(new Date());
```

Comment:
Are you sure this is valid?

`DateFieldProps` type says it can accept `value` with type of `string|undefined`.

Please fix me if I'm not right.

Reply:
Yes but all Zeisslets that I've seen transmit Date and it works correctly. I suppose types should be changed to `string|Date|undefined` but it affects most of the code, so I prefer to leave it as it is.

Reply:
🤔 Can't find any example of using the `Date` instance instead of `string`...

Reply:
Example reworked